tests/micro/arduino/conftest.py

```diff
@@ -48,13 +35,23 @@
 ).resolve()
 
 
+def arduino_micro_devices() -> dict:
+    sys.path.insert(0, str(TEMPLATE_PROJECT_DIR))
+    try:
+        import microtvm_api_server
+    finally:
+        sys.path.pop(0)
+
+    return microtvm_api_server.MICRO_DEVICES
+
+
 def pytest_addoption(parser):
     parser.addoption(
-        "--microtvm-platforms",
```

Comment:
sorry the previous top-level comment was about this flag in both the arduino and zephyr tests